assembly/README.md

```diff
@@ -28,7 +28,7 @@ Pushing to a specific docker registry under a specific account:
 ### Run
 
     docker run -td --name kapua-sql -p 8181:8181 -p 3306:3306 kapua/kapua-sql
-    docker run -td --name kapua-elasticsearch -p 9200:9200 -p 9300:9300 elasticsearch:2.4 -Dcluster.name=kapua-datastore -Ddiscovery.zen.minimum_master_nodes=1
+    docker run -td --name kapua-elasticsearch -p 9200:9200 -p 9300:9300 elasticsearch:5.3.0 -Ecluster.name=kapua-datastore -Ediscovery.zen.minimum_master_nodes=1 -Enetwork.host=0.0.0.0 
```

Comment:
I tried to launch the `kapua-elasticsearch` with this new command, but it seems the command is wrong since I get a 
```
docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"-Ecluster.name=kapua-datastore\": executable file not found in $PATH".
```

What worked for me was
```
docker run -td --name kapua-elasticsearch -p 9200:9200 -p 9300:9300  -e "cluster.name=kapua-datastore" -e "transport.tcp.connect_timeout=30s" -e "transport.ping_schedule=-1" -e "xpack.security.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:5.3.0 
```

I can provide a fix if that's ok to you